Formula/texmaker.rb

```diff
@@ -0,0 +1,61 @@
+class Texmaker < Formula
+  desc "Free cross-platform LaTeX editor since 2003"
+  homepage "https://www.xm1math.net/texmaker/"
+  url "https://www.xm1math.net/texmaker/texmaker-5.1.2.tar.bz2"
+  sha256 "526896f2c1ae561130eec7aae815b9dcda9e8eeb772b6541d0dc94ce91a71044"
+  license "GPL-2.0-only"
+
+  depends_on xcode: :build
+  depends_on "qt"
+
+  on_linux do
+    depends_on "gcc"
+  end
+
+  fails_with gcc: "5"
+
+  patch :p0, :DATA
```

Comment:
Has this patch been upstreamed? We'll probably want to wait for a version where this is included before adding this.

Reply:
There is no public repository. There is just a mail address. I sent a mail with this patch but did not get any response so far. (I wonder a bit, this Qt 5 and Qt 6 docs say that the code should never have worked like this.)

Reply:
Could you follow it up? That makes it even more important that the patch is upstreamed before this formula is added.

Reply:
I got an answer now:

> Thanks for the patch. Will be used in the next release.

Reply:
I further asked when the next release will happen. This is the answer:

> I don't think there will be a new release before end of february 2022.

So it means this PR here has to wait until then?